boards/adafruit/feather_esp32s2/board.yml

```diff
@@ -0,0 +1,22 @@
+boards:
+- name: adafruit_feather
+  full_name: Adafruit Feather ESP32S2
+  vendor: adafruit
+  socs:
+  - name: esp32s2
+  revision:
+    format: "letter"
+    default: "C"
+    revisions:
+    - name: "B"
+    - name: "C"
+- name: adafruit_feather_tft
+  full_name: Adafruit Feather ESP32S2 TFT
+  vendor: adafruit
+  socs:
+  - name: esp32s2
+- name: adafruit_feather_tft_reverse
```

Comment:
these need to be named adafruit_feather_esp32s2_*, Kconfigs will need to be updated also (if you try building for `adafruit_feather` you will see you are actually building for a completely different board)

Reply:
Thanks